packages/components/molecules/f-rating/test/visual/f-rating-singleStar.visual.spec.js

```diff
@@ -0,0 +1,142 @@
+import RatingSingleStar from '../../test-utils/component-objects/f-rating-singleStar.component';
+
+const devices = [
+    'desktop'
+];
+
+devices.forEach(device => {
+    describe('f-rating Single Star - %s - Visual tests', () => {
+        describe('`isSingleStarVariant`', () => {
+            it('should display a single star variant', async () => {
+                // Act
+                await RatingSingleStar.load({ isSingleStarVariant: true });
+
+                // Assert
+                await browser.percyScreenshot('f-rating Single Star - Visual Test for Prop - isSingleStarVariant = true', device);
+            });
+        });
+
+        describe('`starRating`', () => {
+            it('should be displayed', async () => {
+                // Act
+                await RatingSingleStar.load({ isSingleStarVariant: true, starRating: 5 });
+
+                // Assert
+                await browser.percyScreenshot('f-rating Single Star - Visual Test for Prop - starRating = 5', device);
+            });
+        });
+
+        describe('`maxStarRating`', () => {
+            it('should be displayed', async () => {
+                // Act
+                await RatingSingleStar.load({ isSingleStarVariant: true, starRating: 5 });
+
+                // Assert
+                await browser.percyScreenshot('f-rating Single Star - Visual Test for Prop - maxStarRating = 5', device);
+            });
+        });
+
+        describe('`reviewCount`', () => {
+            it('should be displayed', async () => {
+                // Act
+                await RatingSingleStar.load({ isSingleStarVariant: true, reviewCount: 700 });
+
+                // Assert
+                await browser.percyScreenshot('f-rating Single Star - Visual Test for Prop - reviewCount = 700', device);
+            });
+        });
+
+        describe('`starRatingSize`', () => {
+            it('should be displayed at the correct size', async () => {
+                // Act
+                await RatingSingleStar.load({ isSingleStarVariant: true, starRatingSize: 'small' });
+
+                // Assert
+                await browser.percyScreenshot('f-rating Single Star - Visual Test for Prop - starRatingSize = small', device);
```

Comment:
In future it would be good to combine these few tests with `it.each`

Reply:
I did initially try that but as the loop at the top has a `each` attached to it, it caused issues. I'll take a look at resolving that next.